program.js

```diff
@@ -0,0 +1,21 @@
+process.stdin.setEncoding("utf-8");
+process.stdin.on("readable", function() {
+  var input = process.stdin.read();
+  if (input !== null) {
+    var instruction = input.toString().trim();
+    switch (instruction) {
```

Comment:
try to move switch outside the if statement - remember less nesting is better ;)